server/parsec/templates/invitation_mail.html

```diff
@@ -470,11 +473,13 @@
                         </em>
                       </div>
                       <p>
-                        {% if greeter %}
+                        {% if is_user_invitation %}
                         3. Get in touch with <b>{{ greeter }}</b> and follow the next steps on the Parsec client.
-                        {% else %}
+                        {% elif is_device_invitation %}
                         3. Start the invitation process from a device already part of the organization,
                         then follow the steps on the Parsec client.
+                        {% elif is_shamir_recovery_invitation %}
+                        3. Get in touch with <b>{{ greeter }}</b> and follow the next steps on the Parsec client.
```

Comment:
There is one mail sent per greeter ? Does the order of greeter matter ?

Reply:
In this context, the greeter is the user that created the invitation link. That user is part of the recipients so it's a good idea to include their name in the mail, although the claimer will have to contact some of the other recipients too.